### DISCUSSION TITLE
403 Forbidden error showing in local when calling Ted API for Submit and stop e-Form Notice.

### DISCUSSION THREAD

**Discussion by ankur-mercell** [OP-TED/eForms-SDK#796] [Q&A] [answered]
Team, when we calling submit and stop (Ted API ) for e-Form Notice. getting 403 Forbidden status code. its working in dev environment but not working in local. please help out resolve this issue.

below docs we have referring
https://docs.ted.europa.eu/api/endpoints/enotices2-ted-europa-eu-esenders.html

**Comment by pcolast** [comment]
Good Day, 

Is it your first attempt to use Production environment ? Or has it worked for you before ? Could you provide the period when you tried the API calls and received the 403 Forbidden status code.

Thank you.

Stefan

**Comment by ankur-mercell** [comment, marked as answer]
Issue has resolved, token key has changed
